server/src/modules/users/users.dto.ts

```diff
@@ -30,17 +30,14 @@ export class CreateUserDto {
   @IsNotEmpty({
     message: 'The password is required',
   })
-  @Length(8, 30, {
+  @Length(8, 64, {
     message:
-      'The password must be at least 8 but not longer than 30 characters',
+      'The password must be at least 8 but not longer than 64 characters',
```

Comment:
yikes! 64 chars?

Reply:
![image](https://github.com/user-attachments/assets/0cbeb2e4-4596-49e7-87d6-20959e1c9696)
![image](https://github.com/user-attachments/assets/fea251e4-3ee4-40af-bddb-21cb08e887a7)

Reply:
When you use generated passwords like me 30 is nothing.